app/assess/routes.py

```diff
@@ -289,16 +293,27 @@ def application(application_id):
     if flag:
         state.workflow_status = "FLAGGED"
         accounts = get_bulk_accounts_dict([flag.user_id])
-
+        
+    sub_criteria_status_completed = all_status_completed(state)
+    form = AssessmentCompleteForm()
+    if request.method=="POST":
+        requests.post(
+            Config.ASSESSMENT_UPDATE_STATUS.format(
+                application_id=application_id)
+            )
+   
     return render_template(
         "assessor_tasklist.html",
+        sub_criteria_status_completed = sub_criteria_status_completed,
+        form = form,
         state=state,
         application_id=application_id,
         flag=flag,
+        show_prompt= False,
```

Comment:
Is this needed if it's always going to be false?

Reply:
Hmm, i think i may have added it when i was trying to use JS but forgot to remove. Wil remove it